.npmignore

```diff
@@ -6,3 +6,4 @@
 /node-*
 /service-process.pid
 .DS_Store
+package-lock.json
```

Comment:
Please could you not ignore the package lock file, this is a very important file to keep for this project as it ensures we pull in the exact versions of third party polyfills that we have tested on BrowserStack